envoy-control-tests/src/main/kotlin/pl/allegro/tech/servicemesh/envoycontrol/CanaryLoadBalancingTest.kt

```diff
@@ -159,7 +159,17 @@ open class CanaryLoadBalancingTest {
     }
 
     val CallStats.regularHits: Int
-        get() = this.hits(regularContainer)
+        get() = this.hits(regularContainer())
     val CallStats.canaryHits: Int
-        get() = this.hits(canaryContainer)
+        get() = this.hits(canaryContainer())
+
+    open fun envoyControl() = envoyControl
```

Comment:
I guess you exposed these for our internal extensions. Maybe we should rewrite those instead of opening random tests in envoy-control?

Reply:
Extensions are nice but more difficult to override. Maybe we shouldn't override them but just create new test in our internal control-plane.